.travis.yml

```diff
@@ -3,6 +3,7 @@ env:
   - TEST_SUITE=server_ci
   - TEST_SUITE=client_ci
   - TEST_SUITE=continuum-auth_ci
+  - IMAGE_TAG=$TRAVIS_COMMIT
```

Comment:
This doesn't specify the environment, it specifies the _build matrix_ - each build will have one of these values. Stuff like this needs to go in the makefile, not in the `.travis.yml`. Which is why you see an additional job in the build, and presumably the images are not tagged with the correct values.

Reply:
this should be `IMAGE_TAG=$TRAVIS_COMMIT  TEST_SUITE=server_ci`, in 3 variations?